* <grobian@gentoo.org>: /bin/sh is no good (on e.g. Solaris) same for perl

--- common.mk
+++ common.mk
@@ -8,16 +8,17 @@
 
 override PACKAGE  := realpath
 
-PATH            := /usr/bin:/usr/sbin:/bin:/sbin:$(PATH)
+#PATH            := /usr/bin:/usr/sbin:/bin:/sbin:$(PATH)
+SHELL           := @GENTOO_PORTAGE_EPREFIX@/bin/bash
 
 # build abstraction
-install_file    := install -p -o root -g root -m 644
-install_script  := install -p -o root -g root -m 755
-install_dir     := install -d -o root -g root -m 755
+install_file    := install -p -m 644
+install_script  := install -p -m 755
+install_dir     := install -d -m 755
 install_link    := ln -sf
 compress        := gzip -9f
 
-prefix          := /usr
+prefix          := @GENTOO_PORTAGE_EPREFIX@/usr
 etcdir          := /etc/$(PACKAGE)
 bindir          := $(prefix)/bin
 sbindir         := $(prefix)/sbin
@@ -33,11 +34,11 @@
 webcgidir	      := $(prefix)/lib/cgi-bin
 applicationsdir := $(prefix)/share/applications
 
-PERL            := /usr/bin/perl
+PERL            := @GENTOO_PORTAGE_EPREFIX@/usr/bin/perl
 CC              ?= gcc
 CFLAGS          += -Wall -Wextra -Wstrict-prototypes -Wmissing-prototypes -DVERSION='"$(VERSION)"' -DPACKAGE='"$(PACKAGE)"' -DLOCALEDIR='"$(nlsdir)"'
 LDFLAGS         +=
-LIBS            := 
+LIBS            += 
 
 ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
   CFLAGS += -O0
